Packs/AWS-SQS/TestPlaybooks/playbook-AWS-SQS-Test.yml

```diff
@@ -1,31 +1,26 @@
-id: fd93f620-9a2d-4fb6-85d1-151a6a72e46d
+id: AWS - SQS Test Playbook
```

Comment:
you should change all other occurrences in the repo.
specifically the conf.json